src/core/actions/creators.ts

```diff
@@ -20,125 +20,262 @@ namespace ActionCreators {
   // fetch action creators
   /**
    * Makes a request for more refinements for given navigation.
-   * @param  {string}                       navigationId - The navigationId for
-   * the navigation to fetch more refinements against.
-   * @return {Actions.FetchMoreRefinements}              - Action with navigationId.
+   * @param  {Actions.Payload.Fetch.MoreRefinements} options - An object with the navigationId for
+   * the navigation to fetch more refinements against and a request object for override.
+   * @return {Actions.FetchMoreRefinements} - Action with { navigationId, request }.
```

Comment:
This is parsed as Markdown, so `{ navigationId, request }` should be in backticks. Same goes for the others.